### PR TITLE
[Lottie exporter] Add animation to addtion/deletion of Bline points

### PR DESCRIPTION
**Pull request by AnishGG**
Resolves: #858

A long time needed fix for outlines, advanced outlines, and region layer is finally here :smiley:

### CODE REVIEW

**Comment by AnishGG** [comment]
Does anyone know, why are the test scripts not showing up in the checks? The Lottie test script does not run anymore: https://github.com/synfig/synfig/blob/master/autobuild/lottie_test.sh
This script is crucial before merging PRs related to the Lottie exporter.

**Comment by ice0** [comment]
> Does anyone know, why are the test scripts not showing up in the checks? The Lottie test script does not run anymore: https://github.com/synfig/synfig/blob/master/autobuild/lottie_test.sh
> This script is crucial before merging PRs related to the Lottie exporter.

This is because this script was run on Travis CI, but after issues with Travis CI it has not been restored. 
I added new Github CI job for tests (https://github.com/synfig/synfig/pull/2238). Will be merged soon.

File: synfig-studio/plugins/lottie-exporter/common/Activepoint.py [hunk 1]
```diff
@@ -24,9 +24,9 @@ def __init__(self, time, state, priority = 0):
 
         # Convert time to frames
         if time == 'SOT':
-            self.time = settings.lottie_format["ip"]
+            self.time = -32767*512
```
Comment (ice0) [inline synfig-studio/plugins/lottie-exporter/common/Activepoint.py:27]: Maybe make these values constants?

Comment (AnishGG) [reply synfig-studio/plugins/lottie-exporter/common/Activepoint.py]: Resolved ^__^

**Comment by AnishGG** [comment]
@ice0 This PR is now ready :D

**Comment by ice0** [comment]
> Does anyone know, why are the test scripts not showing up in the checks? The Lottie test script does not run anymore: https://github.com/synfig/synfig/blob/master/autobuild/lottie_test.sh
> This script is crucial before merging PRs related to the Lottie exporter.

Travis CI is back and lottie tests also added to Github CI.

Can you rebase so we can check if the tests passed?

**Comment by AnishGG** [comment]
> > Does anyone know, why are the test scripts not showing up in the checks? The Lottie test script does not run anymore: https://github.com/synfig/synfig/blob/master/autobuild/lottie_test.sh
> > This script is crucial before merging PRs related to the Lottie exporter.
> 
> Travis CI is back and lottie tests also added to Github CI.
> 
> Can you rebase so we can check if the tests passed?

Done ^__^

**Review by morevnaproject — APPROVED** [review]

**Comment by morevnaproject** [comment]
Merged! Thank you very much! ^__^

**Pull request merged**